### ISSUE TITLE
RadViz does not plot when transform is called

### ISSUE THREAD

**Issue by naresh-bachwani** [DistrictDataLabs/yellowbrick#793]
**Describe the bug**
I wanted to fit the model on the dataset and obtain its plot by using transform function on their subset. While working with RadViz, I found that it plots the graph when fit method is called. However, it should plot the output, when the transform method is implemented as described in the domentation. Moreover, I did not get the desired result (i.e. plotting) when transform method is called.
I was able to get these results using 'Rank1D,' but 'RadViz' did not work. 

**Output with Rank1D**
![Screenshot (122)](https://user-images.githubusercontent.com/43993586/54885778-f7794100-4ea5-11e9-9dab-1d46a4686f75.png)

**Output with RadViz**
![Screenshot (123)](https://user-images.githubusercontent.com/43993586/54885776-f3e5ba00-4ea5-11e9-8035-ab3376664560.png)

**To Reproduce**
```python
from yellowbrick.features import Rank1D
from yellowbrick.features import RadViz
fig=plt.figure()
visualizer = RadViz(features=features,classes=classes)
visualizer.fit(X, y)                # Fit the data to the visualizer
for i in range(5):
    ax0 = fig.add_subplot(5,1,i+1)
    visualizer.ax=ax0
    visualizer.transform(X[i*100:(i+1)*100])             # Transform the data
        
fig=plt.figure()
visualizer = Rank1D(features=features, algorithm='shapiro')
visualizer.fit(X, y)                # Fit the data to the visualizer
for i in range(5):
    ax0 = fig.add_subplot(5,1,i+1)
    visualizer.ax=ax0
    visualizer.transform(X[i*100:(i+1)*100])             # Transform the data
        
        
```

**Dataset**
I used bikeshare dataset and used features = ['temp','humidity','windspeed','registered','riders'] and 'holiday' as y.

**Comment by lwgray** [comment]
@naresh-bachwani  thanks for opening this issue.  We are just returning from a hiatus and will address this after catching up on our backlog of PRs and Issues.

**Comment by Kautumn06** [comment]
Hi @naresh-bachwani, thank you for your patience as we catchup with our backlog of PRs and Issues! So that I can help you with this, can you please also share with me the code you used to load the bikeshare dataset? Thanks!

**Comment by naresh-bachwani** [comment]
Hi @Kautumn06, the dataset is used just as an example and not to obtain some explicit results. I was working with bikeshare and used one of its categorical columns to produce the output. We can use occupancy or any other dataset.
I was working on issue #507 related to the animation of feature visualizer and noticed that all feature visualizer except RadViz produce an output when transform is called.
The code for producing data is as follows:
```python
data = load_data("bikeshare")
X=data[features]
y=data.holiday
features = ['temp','humidity','windspeed','registered','riders']
classes = ['holiday','notholiday']
```
where load_data is the same function from yellowbrick's example dataset in documentation

**Comment by bbengfort** [comment]
@naresh-bachwani thank you for bringing this up. In order to investigate this, we'll need to remind ourselves of the API for feature visualizers, so I've labeled the issue as technical debt. In the meantime, I'll say that the most common way we use the feature visualizers is through `fit_transform` rather than either `fit` or `transform`. Obviously, we don't want to have our API behave strangely - but do you think that using this method would change your experience, or do you think that you might want to fit the visualizer on different data that you transform on? 

Because this requires an API review, I will say that we probably won't be able to get to it for a little bit. Is this blocking your work on #507?

**Comment by naresh-bachwani** [comment]
I raised this issue only to highlight this bug in the API. As far as #507 is concerned I will try using fit transform method. Thanks for help!!

**Comment by rebeccabilbro** [comment]
@naresh-bachwani @bbengfort I believe this issue has been resolved following the changes made in #973 and for #960. Now neither of these visualizers actually plots until `show()` is called. Can you guys confirm that this issue can be closed?

**Comment by bbengfort** [comment]
@rebeccabilbro @naresh-bachwani I believe that this issue with the inconsistent API has now been fixed, and it's a matter of more deeply researching how to do multi-visualizer/multi-figure drawing. Let's go ahead and close this issue and explore it more specifically if it comes up in a different way!
